docs/evm/evm-local-setup.md

```diff
@@ -0,0 +1,325 @@
+---
+id: evm-local-setup
+title: EVM local setup
+sidebar_label: Local setup
+---
+
+This section covers how to try the NEAR EVM on a local node. Note, this will take longer and require more resources from your computer than using Betanet. This is for those who prefer to develop everything locally. Please note that these instructions are raw at the moment. The goal would be to Dockerize this process, but it's made available for intrepid builders.
+
+## Table of Contents
+
+- [Prerequisites](#prerequisites)
+- Repositories and setup
+  - [Get Rust](#get-rust)
+  - [Set up NEAR node](#set-up-near-node)
+  - [Postgres](#postgres)
+  - [NEAR Contract Helper](#near-contract-helper)
+  - [NEAR CLI](#near-cli)
+  - [Linkdrop smart contract](#near-linkdrop)
+  - [NEAR Wallet](#near-wallet)
+  - [NEAR Explorer](#near-explorer)
+- Pet Shop instructions
+  - [Copying local keys](#copy-key-to-pet-shop)
+  - [Build, migrate, deploy](#localnet-migration)
+- [Notes](#notes)
+- [Troubleshooting](#troubleshooting)
+
+---
+
+### Prerequisites
+
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [Docker](https://docs.docker.com/get-docker/).
+- NodeJS version 12+. We recommend using [a version manager](https://nodejs.org/en/download/package-manager/).
+- [Python 3](https://www.python.org/download/releases/3.0/) and the [pip3 package](https://pip.pypa.io/en/stable/installing/)
```

Comment:
https://www.python.org/download/releases/3.0/ is not a useful link

Reply:
Yes, I searched for a while for a good canonical link. Searched again and didn't find exactly what I was hoping for. Went with https://www.python.org/download